packages/validator/src/util/externalSignerClient.ts

```diff
@@ -229,5 +232,9 @@ function serializerSignableMessagePayload(config: BeaconConfig, payload: Signabl
 
     case SignableMessageType.BLS_TO_EXECUTION_CHANGE:
       return {BLS_TO_EXECUTION_CHANGE: ssz.capella.BLSToExecutionChange.toJson(payload.data)};
+
+    case SignableMessageType.BLOB:
+      // TODO: freetheblobs
```

Comment:
suggest commenting this with `TODO DENEB` prefix so that we don't miss any of these TODOs by just doing a quick search